azurelinuxagent/common/version.py

```diff
@@ -146,6 +159,8 @@ def get_distro():
 DISTRO_CODE_NAME = __distro__[2]
 DISTRO_FULL_NAME = __distro__[3]
 
+LIS_VERSION = get_lis_version()
```

Comment:
why add a new module global instead of just calling the function?

Reply:
For consistency with the existing pattern of code. I can change it.

Reply:
(I actually had written it the way you suggest first, but changed it because it was inconsistent.)

Reply:
Thanks; we are rewriting most of this code since some of the patterns it follows tend to be problematic. Do not worry about consistency.

Reply:
Ok, I personally think it's less consistent now, but it is as you asked.

Reply:
Ah I missed your message 😂